server/src/main/java/com/broadcom/lsp/cobol/core/visitor/VariableDefinitionDelegate.java

```diff
@@ -370,35 +389,41 @@ private void closePreviousStructureIfNeeded(VariableDefinitionContext variable)
 
   private void checkTopElementNumber(VariableDefinitionContext variable) {
     int number = variable.getNumber();
-    if (number == 1 || number == LEVEL_66 || number == LEVEL_77) return;
-    addError(messages.getMessage(NUMBER_NOT_ALLOWED_AT_TOP), variable.getDefinition());
+    if (number == 1 || number == LEVEL_66 || number == LEVEL_77 || number == LEVEL_88) return;
```

Comment:
OPTIONAL:  We have type check now, I believe better to have LEVEL_1  (only for this, it also starts in section A like 66,77,88)

Reply:
Good point. BTW, I didn't notice that 88 is also here - it is not allowed as a top element. Probably, there is some logic violation here.